packages/cw-orch-contract-derive/src/lib.rs

```diff
@@ -163,19 +192,26 @@ pub fn interface(attrs: TokenStream, input: TokenStream) -> TokenStream {
     };
 
     let name = cw_orch_struct.ident.clone();
+    let default_num = if let Some(id_expr) = default_id {
```

Comment:
I don't like setting the id forcefully here. Can we keep the old version if no id was provided and put this new version only if a default id is provided ?

Reply:
That's actually the goal to make it not generate `new` by default so it's handled by the contract maintainer

Reply:
Oh, hmm, ok ! Adds an extra argument that has nothing to do with messages, but that could work

Reply:
> Oh, hmm, ok ! Adds an extra argument that has nothing to do with messages, but that could work

We totally could change syntax of it like 
```rust
#[interface(InstantiateMsg, ExecuteMsg, QueryMsg, MigrateMsg)]
#[contract_id=CONTRACT_ID]
```